keyboards/chalice/rules.mk

```
@@ -0,0 +1,15 @@
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
+
+# Build Options
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```

There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```